static/js/stream_events.js

```diff
@@ -168,6 +168,7 @@ export function mark_unsubscribed(sub) {
 
     if (narrow_state.is_for_stream_id(sub.stream_id)) {
         message_lists.current.update_trailing_bookend();
+        unread_ui.hide_mark_as_read_turned_off_banner();
```

Comment:
Hmm, but do you still see this banner after reloading the page, or does the check for whether to display it in the first case look at whether you're subscribed or not?

Reply:
I don't see this banner after reloading the page. It still has the same interactions as CZO. I'm not quite understanding the later parts, could you go a bit more in-depth about it?

Reply:
OK, I think the answer is that we only call `notify_messages_remain_unread` if there are unread messages, and there can't be in a stream you're not subscribed to, so it won't recur.